flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/common/utils/PulsarTransactionUtils.java

```diff
@@ -60,6 +60,8 @@ public static Transaction createTransaction(PulsarClient pulsarClient, long time
     /**
      * This is a bug in original {@link TransactionCoordinatorClientException#unwrap(Throwable)}
      * method. Pulsar wraps the {@link ExecutionException} which hides the real execution exception.
+     *
+     * <p>This bug should be fixed after the 2.10.1 release. We just keep this for safety.
```

Comment:
Out of curiosity, can you add the link of Pulsar PR in this thread? When I check the master branch of pulsar, it seems the method `TransactionCoordinatorClientException.unwrap` doesn't change.

Reply:
It was fixed in https://github.com/apache/pulsar/pull/14214. Pulsar will reclusively call the `unwrap` method.

Reply:
```suggestion
     * <p>This bug should be fixed after the 2.10.0 release. We just keep this for safety.
```